drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTableViewImpl.java

```diff
@@ -0,0 +1,640 @@
+                                                                                                         presenter,
+                                                                                                         eventBus,
+                                                                                                         column,
+                                                                                                         true,
+                                                                                                         !access.isEditable() );
+        popup.show();
+    }
+
+    @Override
+    public void newExtendedEntryActionInsertColumn() {
+        final ActionInsertFactCol52 column = new ActionInsertFactCol52();
+        final ActionInsertFactPopup popup = new ActionInsertFactPopup( model,
+                                                                       oracle,
+                                                                       presenter,
+                                                                       new ActionColumnCommand() {
+                                                                           public void execute( final ActionCol52 column ) {
+                                                                               presenter.appendColumn( column );
+                                                                           }
+                                                                       },
+                                                                       column,
+                                                                       true,
+                                                                       !access.isEditable() );
+        popup.show();
+    }
+
+    @Override
+    public void newLimitedEntryActionInsertColumn() {
+        final LimitedEntryActionInsertFactCol52 column = new LimitedEntryActionInsertFactCol52();
+        final ActionInsertFactPopup popup = new ActionInsertFactPopup( model,
+                                                                       oracle,
+                                                                       presenter,
+                                                                       new ActionColumnCommand() {
+                                                                           public void execute( final ActionCol52 column ) {
+                                                                               presenter.appendColumn( column );
+                                                                           }
+                                                                       },
+                                                                       column,
+                                                                       true,
+                                                                       !access.isEditable() );
+        popup.show();
+    }
```

Comment:
The two methods above are the same except for the first line...perhaps reuse the code instead?